cpp/src/arrow/compute/exec/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_arrow_compute_test(union_node_test PREFIX "arrow-compute")
 add_arrow_compute_test(util_test PREFIX "arrow-compute")
 
 add_arrow_benchmark(expression_benchmark PREFIX "arrow-compute")
+add_arrow_benchmark(expression_execution_benchmark PREFIX "arrow-compute")
```

Comment:
I'm not sure it's useful to create a separate executable for this. Simply add the benchmark to the existing expression benchmark file?

Reply:
Removed the file and moved the benchmark to the expression benchmark file.